build.gradle

```diff
@@ -42,3 +65,60 @@ swaggerSources {
 compileJava.dependsOn swaggerSources.ssc.code
 sourceSets.main.java.srcDir "${swaggerSources.ssc.code.outputDir}/src/main/java"
 sourceSets.main.resources.srcDir "${swaggerSources.ssc.code.outputDir}/src/main/resources"
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            pom {
+                groupId = project.group
+                artifactId = project.name
+                name = project.name
+                version = project.version
+                description = project.description
+                url = 'https://github.com/fortify/ssc-restapi-client'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
```

Comment:
Evgeny, our project is MIT-licensed, it's something that most of Jenkins plugins use. Is it possible to post MIT-licensed jars to Maven?

Reply:
As the original ssc-restapi-client was MIT-licensed, I restored that license to this branch